xarray/backends/cfgrib_.py

```diff
@@ -38,7 +43,6 @@ class CfGribDataStore(AbstractDataStore):
     """
 
     def __init__(self, filename, lock=None, **backend_kwargs):
-        import cfgrib
```

Comment:
If the import is later in the code I would suggest not to remove this additional import. Technically is not needed, but it is very confusing to rely on an import statement that is later in the code.

Reply:
I agree